torch/csrc/utils/cuda_lazy_init.h

```diff
@@ -1,11 +1,23 @@
 #pragma once
 
-// It initially lies in torch/csrc/cuda, but to unconditionlly compile it
-// we have to put it here.
+// cuda_lazy_init() is always compiled, even for CPU-only builds.
+// Thus, it does not live in the cuda/ folder.
 
 namespace torch {
 namespace utils {
 
+// The INVARIANT is that this function MUST be called before you attempt
```

Comment:
This doesn't square with https://github.com/pytorch/pytorch/blob/64fc00fcd21073c7c8119a08da782d57c08510a1/tools/autograd/templates/python_torch_functions_dispatch.h#L31-L35 -- either the message is wrong or that function is bogus.

Reply:
you're right, the function looks bogus.